ARGO/incoming_handler.sh

```diff
@@ -65,6 +65,17 @@ handle_additions() {
     done
 }
 
+
+# lock processing of talend trigger to prevent simultanous talend runs
+lock() {
+    exec 200>$DATA_SERVICES_TMP_DIR/argo_pid.lock
+
+    flock -n 200 \
```

Comment:
It would be great to have a space to indicate whether it's exec called with an argument of 200, or exec redirected using the 200 filedescriptor. Eg.
```
exec 200> $DATA_SERVICES_TMP_DIR/argo_pid.lock
```